cva6/cva6-32bit.patch

```diff
@@ -1,13 +0,0 @@
-diff --git a/include/riscv_pkg.sv b/include/riscv_pkg.sv
```

Comment:
Is this file supposed to be committed?

Reply:
`cva6-32bit.patch` is no more required and therefore was deleted.
`include/riscv_pkg.sv` is the patched file located in cva6 repository where it is still there.